File: Modul 4/C/PRAK404-23108317310005-MuhammadNurwahyudiAdhitama.c
#include <stdio.h>

int main() {
    int pilihan;
    float nilaiPertama, nilaiKedua;

    while (1) {
        printf("Pilih program:\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");
        printf("Masukkan Pilihan: ");
        scanf("%d", &pilihan);

        if (pilihan == 5) {
            printf("Terimakasih, telah menggunakan kalkulator Muhammad Nurwahyudi Adhitama\n");
            break;
        }

        if (pilihan < 1 || pilihan > 4) {
            printf("Input anda salah, Silakan coba lagi.\n");
            continue;  
        }

        printf("Masukkan nilai pertama: ");
        scanf("%f", &nilaiPertama);
        printf("Masukkan nilai kedua: ");
        scanf("%f", &nilaiKedua);

        switch (pilihan) {
            case 1:
                printf("Hasil Penjumlahan antara %.2f dan %.2f adalah %.2f\n", nilaiPertama, nilaiKedua, nilaiPertama + nilaiKedua);
                break;
            case 2:
                printf("Hasil Pengurangan antara %.2f dan %.2f adalah %.2f\n", nilaiPertama, nilaiKedua, nilaiPertama - nilaiKedua);
                break;
            case 3:
                printf("Hasil Perkalian antara %.2f dan %.2f adalah %.2f\n", nilaiPertama, nilaiKedua, nilaiPertama * nilaiKedua);
                break;
            case 4:
                if (nilaiKedua != 0) {
                    printf("Hasil Pembagian antara %.2f dan %.2f adalah %.2f\n", nilaiPertama, nilaiKedua, nilaiPertama / nilaiKedua);
                } else {
                    printf("Pembagian oleh nol tidak valid.\n");
                }
                break;
        }
    }

    return 0;
}

File: Modul 4/Python/PRAK404-23108317310005-MuhammadNurwahyudiAdhitama.py
while True:
    print("Pilih program:")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")
    
    pilihan = int(input("Masukkan Pilihan: "))

    if pilihan == 5:
        print("Terimakasih, telah menggunakan kalkulator Muhammad Nurwahyudi Adhitama")
        break

    if pilihan < 1 or pilihan > 4:
        print("Input anda salah, Silakan coba lagi.")
        continue

    nilaiPertama = float(input("Masukkan nilai pertama: "))
    nilaiKedua = float(input("Masukkan nilai kedua: "))

    if pilihan == 1:
        print(f"Hasil Penjumlahan antara {nilaiPertama:.2f} dan {nilaiKedua:.2f} adalah {nilaiPertama + nilaiKedua:.2f}")
    elif pilihan == 2:
        print(f"Hasil Pengurangan antara {nilaiPertama:.2f} dan {nilaiKedua:.2f} adalah {nilaiPertama - nilaiKedua:.2f}")
    elif pilihan == 3:
        print(f"Hasil Perkalian antara {nilaiPertama:.2f} dan {nilaiKedua:.2f} adalah {nilaiPertama * nilaiKedua:.2f}")
    elif pilihan == 4:
        if nilaiKedua != 0:
            print(f"Hasil Pembagian antara {nilaiPertama:.2f} dan {nilaiKedua:.2f} adalah {nilaiPertama / nilaiKedua:.2f}")
        else:
            print("Pembagian oleh nol tidak valid.")
